packages/web-runtime/package.json

```diff
@@ -16,6 +16,7 @@
     "fuse.js": "^6.5.3",
     "lodash-es": "^4.17.21",
     "luxon": "^2.3.0",
+    "marked": "^4.0.12",
```

Comment:
By adding this here, you should be able to remove the (now duplicate) dependency in the markdown extension itself:)

Reply:
shouldn't extensions be independent ?

Reply:
As long as we have the extensions in the monorepo I'm fine with maintaining dependencies centralized if they are used in multiple packages. That's what we're already doing, so I'm fine with what Pascal proposed. We'll have to have a close look at dependencies anyway when we split out extensions at some point.